.travis.yml

```diff
@@ -18,6 +30,12 @@ matrix:
               TOXENV=py3
               PYTEST_ADDOPTS=-v  # List all tests run by pytest
         - if: NOT branch =~ /^ubuntu\//
+          cache:
+              - directories:
+                  - lxd_images
+          before_cache:
+              # Copy lxd image files into the directory Travis will cache
+              - sudo find /var/snap/lxd/common/lxd/images/ -type f -exec cp {} lxd_images/ \;
```

Comment:
Will we risk saturating some sort of space cap by always caching all images and pulling them forward in travis on all builds or do we need a way to retire old lxd_images that are no longer applicable? Could we limit this caching to images that were pulled down during the same day or week?

Reply:
That's a great point. I don't think we need to worry about a cap, exactly, but it's certainly the case that the time we save by caching will be reduced the larger that cache gets (because it still needs to be pulled down over _a_ network).

Travis [expires caches after a certain point](https://docs.travis-ci.com/user/caching/#caches-expiration), but I think that's reset if _anything_ in the cache has changed, so we are likely to always keep caches alive, at least in `master`.

I think a day or week would be too short: cloud images generally release on a three week cadence. However, we do only pull down a single image as part of the Travis run, so where currently we copy all the lxd images back into `lxd_images`, we could could instead copy only the most _recent_ image back (and clear out the old images).

Does that sounds reasonable?